yarn.lock

```diff
@@ -14473,6 +14559,31 @@ react-native-windows@^0.62.1:
     uuid "^3.3.2"
     xml-parser "^1.2.1"
 
+react-native-windows@^0.63.0:
```

Comment:
It doesn't look like `react-native-windows`@0.62 is removed? Do you know what's pulling it in?

Reply:
Good catch. Looks like it's app/fluent-tester, our shared JS package the test apps use.

Reply:
I'm gonna move all of that package's react-native dependencies to be peer dependencies...

Reply:
OK, there should be no more references to 0.62 now